test/plugins/test_art.py

```diff
@@ -1019,6 +1019,21 @@ def test_percent(self):
 def suite():
     return unittest.TestLoader().loadTestsFromName(__name__)
 
+def cleanup_tmp_files():
+    # List all files and directories in /tmp that start with 'tmp'
+    temp_files = [f for f in os.listdir('/tmp') if f.startswith('tmp')]
+    
+    # Remove each file or directory
+    for f in temp_files:
+        temp_path = os.path.join('/tmp', f)
+        if os.path.isdir(temp_path):
+            shutil.rmtree(temp_path)
+        else:
+            os.remove(temp_path)
+
 
 if __name__ == "__main__":
     unittest.main(defaultTest="suite")
+    
+    # Cleanup temporary files
```

Comment:
I see that this cleanup only runs when this script is called directly

```python
if __name__ == "__main__":
```

Since we use `pytest` to run tests, I think this logic may not run. Have you tried running, say, `pytest test/plugins/test_art.py` and checking whether the files have been removed?

Reply:
Thank you for the quick response. Let me review and get back to you.